.github/workflows/export.yml

```diff
@@ -17,18 +18,25 @@ jobs:
       AWS_S3_ENDPOINT: ${{ secrets.DO_S3_ENDPOINT }}
       AWS_ACCESS_KEY_ID: ${{ secrets.DO_ACCESS_KEY_ID }}
       AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
-      AWS_S3_BUCKET: edm-recipes
+      AWS_S3_LIBRARY_BUCKET: edm-recipes
```

Comment:
I added this because we export to edm-publishing space as well as edm-recipes. This isn't necessary as the edm-recipes path ended up living in the `export_do` bash case and not the workflow .yml. Even if not necessary I think it makes it less confusing to a developer who could get the DO spaces mixed up. If this doesn't add clarity for others happy to change it back